src/harness/testcases/WINNF_FT_S_SCS_testcase.py

```diff
@@ -252,3 +256,218 @@ def test_WINNF_FT_S_SCS_15(self,config_filename):
 
     # Check registration response
     self.assertEqual(response['response']['responseCode'],104)
+
+  def generate_SCS_17_default_config(self, filename):
+    """Generates the WinnForum configuration for SCS_17. """
+    # Create the configuration for short lived invalid client cert/key,wait timer
+
+    # Get current working directory
+    cwd = os.getcwd()
```

Comment:
Is it really necessary to switch dirs?

Reply:
By   switching directory it will create certificate in certs directory. This the   same directory structure can be used in the new script and the it can also be   executed independently from the certs directory if needed.